app/components/cases/new-subcase-form.hbs

```diff
@@ -0,0 +1,191 @@
+                  <Textarea
+                    aria-labelledby="name-subcase"
+                    rows="4"
+                    class="auk-textarea"
+                    @value={{mut this.title}}
+                  />
+                </AuFormRow>
+                <AuFormRow>
+                  <Auk::Label>{{t "subcase-type-dropdown"}}</Auk::Label>
+                  <Utils::ModelSelector
+                    @modelName="subcase-type"
+                    @searchField="label"
+                    @sortField="label"
+                    @displayField="label"
+                    @selected={{this.subcaseType}}
+                    @onChange={{this.selectSubcaseType}}
+                  />
+                </AuFormRow>
+                <AuFormRow>
+                  <Auk::Label>{{t "subcase-name"}}</Auk::Label>
+                  <div class="auk-o-flex auk-o-flex--vertical-center auk-o-flex-gap--small au-u-maximize-width">
+                    {{#if (not this.isEditing)}}
+                      <Utils::ModelSelector
+                        class="auk-u-maximize-width"
+                        @modelName="shortcut"
+                        @searchField="label"
+                        @sortField="label"
+                        @filter={{this.filter}}
+                        @selected={{this.selectedShortcut}}
+                        @onChange={{this.selectSubcaseName}}
+                      as |model|>
+                        {{capitalize model.label}}
+                      </Utils::ModelSelector>
+                      <AuButton
+                        @skin="naked"
+                        @icon="pencil"
+                        @hideText={{true}}
+                        {{on "click" this.toggleIsEditing}}
+                      >
+                        {{t "edit"}}
+                      </AuButton>
+                    {{else}}
+                      <AuInput
+                        @width="block"
+                        @value={{this.subcaseName}}
+                        id="subcaseNameId"
+                      />
+                      <AuButton
+                        @skin="naked"
+                        @icon="x"
+                        @hideText={{true}}
+                        {{on "click" this.toggleIsEditing}}
+                      >
+                        {{t "cancel"}}
+                      </AuButton>
+                    {{/if}}
+                  </div>
+                </AuFormRow>
+              </div>
+            </Utils::LoadableContent>
+          </Auk::Panel::Body>
+        </Auk::Panel>
+      </div>
+    </div>
+  </div>
+  <AuToolbar
+    @reverse={{true}}
+    @size="large"
+    @border="top"
+    as |Group|>
+    <Group>
+      <AuLink
+        @skin="button-secondary"
+        @route="cases.case.subcases"
+      >
+        {{t "cancel"}}
+      </AuLink>
+      <AuButton
+        @skin="primary"
+        @disabled={{this.saveCase.isRunning}}
```

Comment:
Not a task, this doesn't do anything. Should replace `saveCase` with a task instead.